### ISSUE TITLE
Make sure students must ignore critical number outside interval in EVT

### ISSUE THREAD

**Issue by heburson** [StevenClontz/checkit-calculus#6]
Currently, on some EVT problems, if students plug in both critical numbers, the one outside the interval won't give a max or min so it can be hard to tell when grading if students know that they need to ignore it or just lucked out that it did not change the answer.  The generator could be improved by insuring that the critical number outside the interval gives a minimum or maximum function value.

**Comment by StevenClontz** [comment]
Yeah this was a known deficiency when I wrote the question. I purposely made sure there was a red herring, but didn't ensure it always triggered. I'll try to include this in a winter update that merges in improvements from https://stevenclontz.github.io/cal1-tbil-2021-checkit/#/bank/ as well
